webapp/app/components/test-item/template.hbs

```diff
@@ -22,6 +22,11 @@
 </div>
 
 <div class="main-info">
+  {{#if test.is_starred}}
+    <i class="fa fa-star" {{action "toggle_starred"}}></i>
```

Comment:
Please add the `add-margin-right` class here

Reply:
I think this calls for a component, and the color of the star should be `orange`...